docs-v2/content/en/docs/pipeline-stages/deployers/cloudrun.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Google Cloud Run"
+linkTitle: "Google Cloud Run"
+weight: 20
```

Comment:
Change weight to 60. 

The weight corresponds to the order in the left hand menu.